### DISCUSSION TITLE
Why did I get different answers for the same test case? However, I seem to be unable to find the normalization factor.

### DISCUSSION THREAD

**Discussion by lazzy-dog** [The-OpenROAD-Project/OpenROAD#5146] [Q&A]
I used a test case named "gcd", and I obtained the following results ：
  using the Python APIs provided by OpenROAD: 
      dynamic power is 0.005040252273342433, 
      static power is 2.3204001389665052e-05, 
      WNS is -4.4188694e-10, 
      TNS is -7.925274e-08.
 However, the results I obtained through OpenROAD are as follows: 
      dynamic power is 0.005039...,
      static power is 2.322e-05, 
      WNS is -0.38, 
      TNS is -9.60.
   The power consumption on both sides is the same, but there is a significant timing difference. I believe this is the result of normalization done by OpenROAD. So, I called OpenROAD's GUI and found that the ArrivalTime value on 710/CK is 0.067 (Just like the first picture and the second picture ). At the same time, I outputted the same results using the API (without normalization, k=1e-10). However, as the signal moves (for example, from _520_A to _520_ZN), I found that they do not follow the aforementioned rule of k=1e-10. So, my question is, how should I adjust my coefficient? Or is there a better way, can I set the weight within OpenROAD GUI to 1?   

Looking forward to your reply.
Sincerely

![D2FFDB88CE1BF22BC5DE08D39B9755ED](https://github.com/The-OpenROAD-Project/OpenROAD/assets/62580604/5c3ac037-dc8d-4612-8e31-b15e1d033e69)
![20A6CD0A9C5751380AB72741D5AFA578](https://github.com/The-OpenROAD-Project/OpenROAD/assets/62580604/ebfe7e5b-d3b5-409c-b840-4b200ccf0b5b)
![53DF643B8E4AE90DE9A44B0D235B4FB0](https://github.com/The-OpenROAD-Project/OpenROAD/assets/62580604/9e148696-c975-4148-94e7-b7fa77b7f043)

**Comment by maliberty** [comment]
Please provide an executable test case demonstrating this.

**Reply by lazzy-dog** [reply]
Thank you, I packaged a minimal running subsystem with a README file inside.
https://drive.google.com/drive/folders/1_h9jzw8hYpWP3LcAvXDL4-FXRBEbDMCv?usp=sharing
